### ISSUE TITLE
Add second dimension

### ISSUE THREAD

**Issue by mjf** [aravindballa/gatsby-theme-andy#34]
I would like to ask how to use `content/<subdirectory>` and how to reference content in it? It seems to me that only the `content/` directory itself is sourced and processed. I use subdirectories for organizing my notes into areas (i.e. `content/postgres/...`, `content/kernel/...` etc.) Of course I would also like to add subareas (`content/kernel/bpf/...`). I think in a way naming of my notes often overlaps in many areas and I think it is not so uncommon (especially if you are sysadmin). Assuming that default Markdown page is `about.md` my idea is to have the posibility to write:

```markdown
My notes about [[postgres/|PostgreSQL]] are [[interesting]].
```

Which will create a link to `content/postgres/about.md` with title "PostgreSQL" (note the slash "/" which distinquishes from some `content/postgres.md` that may also exist) and some "normal" link to `content/interesting.md`.

(Also, the Wiki syntax `[[link|Title]]` does not seem to work although it would be very useful if it would.)

In other words, I miss the second dimension for my notes with this project and I also can not find some good workaround how to achieve it. Now we have just "single drawer" for the zettlekasten, but mostly you have many drawers and it's also quite common you reference from one "drawer" to another. Filesystem hierarchy is the most elegant way of how to achieve it, IMHO.

**Comment by nishu** [comment]
I am facing a similar issue.
